MdeModulePkg/Bus/Pci/XhciDxe/XhciSched.h

```diff
@@ -1462,7 +1462,8 @@ XhcCreateUrb (
 VOID
 XhcFreeUrb (
   IN USB_XHCI_INSTANCE  *Xhc,
-  IN URB                *Urb
+  IN URB                *Urb,

```

Comment:
The documentation directly above this function needs to be updated to reflect tohe new parameter